clang/include/clang/AST/ExprConcepts.h

```diff
@@ -81,10 +82,21 @@ class ConceptSpecializationExpr final : public Expr, public ConceptReference {
 
   static ConceptSpecializationExpr *
   Create(const ASTContext &C, ConceptDecl *NamedConcept,
+         const ASTTemplateArgumentListInfo *ArgsAsWritten,
          ImplicitConceptSpecializationDecl *SpecDecl,
          const ConstraintSatisfaction *Satisfaction, bool Dependent,
          bool ContainsUnexpandedParameterPack);
 
+  static ConceptSpecializationExpr *
+  Create(const ASTContext &C, ConceptDecl *NamedConcept,
+         ImplicitConceptSpecializationDecl *SpecDecl,
+         const ConstraintSatisfaction *Satisfaction, bool Dependent,
+         bool ContainsUnexpandedParameterPack) {
```

Comment:
The test is complaining that the symbol `_ZN5clang25ConceptSpecializationExpr6CreateERKNS_10ASTContextEPNS_11ConceptDeclEPNS_33ImplicitConceptSpecializationDeclEPKNS_22ConstraintSatisfactionEbb` was removed. Running that through a demangler I get `clang::ConceptSpecializationExpr::Create(clang::ASTContext const&, clang::ConceptDecl*, clang::ImplicitConceptSpecializationDecl*, clang::ConstraintSatisfaction const*, bool, bool)` 

It seems like maybe the test didn't pick up this change?

Reply:
I wonder if the problem is that it isn't USED anywhere, and since it is inlined, it is never being emitted?  Do we need to force it to be emitted in some way (via referencing it somehow, or making it an out-of-line definition?)?

Reply:
Maybe we need to use the LLVM_EXTERNAL_VISIBILITY macro.